#!/usr/bin/env python

PACKAGE='jsk_perception'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("hist_size", int_t, 0, "the number of bins", 10, 1, 255)
gen.add("min_value", double_t, 0, "min value of histogram", 0.0, 0.0, 1000.0)
gen.add("max_value", double_t, 0, "max value of histogram", 256.0, 0.0, 1000.0)

exit(gen.generate(PACKAGE, "jsk_perception", "SingleChannelHistogram"))
